paddle/fluid/operators/scale_op.cc

```diff
@@ -47,9 +46,9 @@ Scale operator
 
 $$Out = scale*X$$
 )DOC");
-    AddAttr<AttrType>("scale",
-                      "(float, default 1.0)"
-                      "The scaling factor of the scale operator.")
+    AddAttr<float>("scale",
```

Comment:
Why only support `float`?

Reply:
Because the Attr is a static variable, we cannot select its type based the input, and I think the attribute be float is enough.